lib/error.js

```diff
@@ -94,5 +98,6 @@ module.exports = {
   MongoError,
   MongoNetworkError,
   MongoParseError,
-  MongoTimeoutError
+  MongoTimeoutError,
+  mongoErrorContext
```

Comment:
just to continue my reign of nitpicky terror: can we rename this to be more self explanatory? `MongoErrorContextSymbol` or something similar?

Reply:
ok, I'll make it `mongoErrorContextSymbol`, since it is not a constructor 😉